sale_order_line_date/README.rst

```diff
@@ -14,12 +14,12 @@ of the lines change to match.
 Usage
 =====
 
-Create a Quotation or a Sales Order and fill the requested date in the sale
+Create a Quotation or a Sales Order and it fills the requested date in the sale
```

Comment:
@lreficent I did not notice of that. This Pull Request must be fixed then. The behavior in previous versions is like the current readme describes, the changes in the requested date in the sale order affect all the lines. I'll check it again